<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SttReplayWaveformEditWidget</class>
 <widget class="QWidget" name="SttReplayWaveformEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>517</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="m_rb_WaveEdit">
            <property name="text">
             <string>原波形编辑</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="m_rb_InsertWave">
            <property name="text">
             <string>插入常态波形</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="m_pb_Replay">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>波形复归</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_6">
               <item row="0" column="0">
                <widget class="QGroupBox" name="m_Insert_GroupBox">
                 <property name="title">
                  <string>插入波形</string>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_3">
                  <item row="0" column="0">
                   <layout class="QGridLayout" name="m_Insert_Layout">
                    <item row="3" column="2">
                     <widget class="QFloatLineEdit" name="m_edt_InsertCycNum"/>
                    </item>
                    <item row="1" column="1">
                     <spacer name="horizontalSpacer">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeType">
                       <enum>QSizePolicy::Fixed</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>20</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item row="3" column="0">
                     <widget class="QLabel" name="label_3">
                      <property name="text">
                       <string>循环次数：</string>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="0">
                     <widget class="QLabel" name="label_2">
                      <property name="contextMenuPolicy">
                       <enum>Qt::NoContextMenu</enum>
                      </property>
                      <property name="text">
                       <string>终止时刻（s）：</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="2">
                     <widget class="QFloatLineEdit" name="m_edt_InsertBegin"/>
                    </item>
                    <item row="1" column="0">
                     <widget class="QLabel" name="m_lb_InsertBegin">
                      <property name="minimumSize">
                       <size>
                        <width>80</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>起始时刻（s）：</string>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="2">
                     <widget class="QFloatLineEdit" name="m_edt_InsertEnd"/>
                    </item>
                    <item row="0" column="2">
                     <widget class="QComboBox" name="m_Cbox_InsertType"/>
                    </item>
                    <item row="0" column="0">
                     <widget class="QLabel" name="label">
                      <property name="text">
                       <string>插入波形参考：</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QGroupBox" name="m_InsertNomal_GroupBox">
                 <property name="title">
                  <string>插入常态波形</string>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_4">
                  <item row="0" column="0">
                   <layout class="QGridLayout" name="m_Insert_Layout_3">
                    <item row="2" column="2">
                     <widget class="QFloatLineEdit" name="m_edt_InsertNormaloutTime"/>
                    </item>
                    <item row="0" column="0">
                     <widget class="QLabel" name="label_8">
                      <property name="minimumSize">
                       <size>
                        <width>0</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>电压（V）：</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="2">
                     <widget class="QFloatLineEdit" name="m_edt_InsertNormalVal"/>
                    </item>
                    <item row="1" column="2">
                     <widget class="QFloatLineEdit" name="m_edt_InsertNormalCur"/>
                    </item>
                    <item row="1" column="0">
                     <widget class="QLabel" name="label_5">
                      <property name="text">
                       <string>电流（A）：</string>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="0">
                     <widget class="QLabel" name="label_7">
                      <property name="text">
                       <string>输出时间（s）：</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="1">
                     <spacer name="horizontalSpacer_3">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeType">
                       <enum>QSizePolicy::Fixed</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>20</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QGroupBox" name="m_DeleteWave_GroupBox">
                 <property name="title">
                  <string>删除波形</string>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_5">
                  <item row="0" column="0">
                   <layout class="QGridLayout" name="m_Insert_Layout_2">
                    <item row="1" column="2">
                     <widget class="QFloatLineEdit" name="m_edt_DeleteEnd"/>
                    </item>
                    <item row="1" column="0">
                     <widget class="QLabel" name="label_4">
                      <property name="text">
                       <string>终止时刻（s）：</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="0">
                     <widget class="QLabel" name="label_6">
                      <property name="minimumSize">
                       <size>
                        <width>0</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>16777215</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>起始时刻（s）：</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="2">
                     <widget class="QFloatLineEdit" name="m_edt_DeleteBegin"/>
                    </item>
                    <item row="0" column="1">
                     <spacer name="horizontalSpacer_2">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeType">
                       <enum>QSizePolicy::Fixed</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>20</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="1" column="1">
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QPushButton" name="m_pb_delete">
                   <property name="maximumSize">
                    <size>
                     <width>100</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>删除</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="m_pb_Insert">
                   <property name="maximumSize">
                    <size>
                     <width>100</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>插入</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QFloatLineEdit</class>
   <extends>QLineEdit</extends>
   <header>../Module/UI/Module/CommonCtrl_QT/QFloatLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
